ProjFS.Mac/PrjFSKext/PrjFSKext.exp

```diff
@@ -0,0 +1,18 @@
+# Symbols that should be exported (public) by the linker. * is wildcard.
```

Comment:
I guess now that we haven't got any templated code left in this PR, it's a weird PR to include it in. Still not a bad idea for general symbol hygiene. I had it in this PR because the original code contained templates, and template instances generate tons of symbols which junk up the symbol tables.

Reply:
Got it, thanks! I think I would prefer to remove the exported symbols for now (at least as part of this PR) until we have templates that will bloat our symbols. I don't feel super strongly about this however.